incorrect_erc20_interface/README.md

```diff
@@ -0,0 +1,21 @@
+## Incorrect erc20 interface
```

Comment:
I don't know if this makes sense but should there be a generalized rule that all interfaces should be correctly configured to return the correct value? This same type of bug could occur in cases where a developer uses a custom or non ERC-20/721 interface?